src/Morphir/IR/SDK.elm

```diff
@@ -64,6 +65,7 @@ packageSpec =
             , ( [ [ "rule" ] ], Rule.moduleSpec )
             , ( [ [ "decimal" ] ], Decimal.moduleSpec )
             , ( [ [ "int" ] ], Decimal.moduleSpec )
+            , ( [ [ "number" ] ], Number.moduleSpec )
```

Comment:
This was omitted accidentally before in resulted in the number type not being usable.